app/controllers/frontend_controller.rb

```diff
@@ -42,7 +41,7 @@ class FrontendController < ApplicationController
 
   layout :pick_buyer_or_provider_layout
 
-  helper_method :quickstarts_presenter, :masthead_data, :alert_type?
+  helper_method :quickstarts_presenter, :sudo
```

Comment:
`masthead_data` doesn't exist, and `sudo` was on another `helper_method` definition - no need to have two.